### ISSUE TITLE
Unexpected source: state=3 source=27713 action=1 (src/transports/inproc/cinproc.c:223)

### ISSUE THREAD

**Issue by drewcrawford** [nanomsg/nanomsg#287]
May be related to #286 

The following test case:

```python
#!python3
import nanomsg
from time import sleep
s = nanomsg.Socket(nanomsg.REP)
s._set_recv_timeout(2)
s.bind("inproc://test")

c = nanomsg.Socket(nanomsg.REQ)
c.connect("inproc://test")

try:
	s.recv()
except Exception as ex:
	print (ex)
s.close()
sleep(2)
s = nanomsg.Socket(nanomsg.REP)
s.bind("inproc://test")
s.close()
```

Using latest nanomsg-python and latest nanomsg consistently produces output

```
Resource temporarily unavailable
Unexpected source: state=3 source=27713 action=1 (src/transports/inproc/cinproc.c:223)
Abort trap: 6
```

The "Resource temporarily unavailable" line is not unexpected since the test intentionally lets the timeout run down.  However the assertion failure is definitely unexpected.

**Comment by marchon** [comment]
I do not believe this is an actionable item except for adding this information to a FAQ

**Comment by gdamore** [comment]
I'm not even sure this is a FAQ item.  You can't bind two workers, programs, etc. to the same interface.  That's true for TCP, IPC, and inproc.  nn_bind() checks that the address being bound is actually available, and is doing the right thing here -- the expected thing if you grok bind().

Closing.

**Comment by baxtersa** [comment]
Just an FYI, this appears to be the same issue I was seeing in the crash for #411.  This specific issue should be fixed by pull request #416  if anyone is still interested and wants to check it out.
